chore: remove crossplane gcp providers

srishtih opened the pull request.

Commit:
Builds crossplane gcp providers

Signed-off-by: Srishti Hegde <srishti-hegde-work-v2@chainguard-workstations.iam.gserviceaccount.com>

Commit:
lints manifest

Signed-off-by: Srishti Hegde <srishti-hegde-work-v2@chainguard-workstations.iam.gserviceaccount.com>

Commit:
Correct key-value mapping of compute and storage

Signed-off-by: Srishti Hegde <srishti-hegde-work-v2@chainguard-workstations.iam.gserviceaccount.com>

srishtih commented:
This PR has now been moved.

srishtih committed:
withdraws apks

Signed-off-by: Srishti Hegde <srishti.hegde@chainguard.dev>

srishtih committed:
Merge branch 'main' into feat/gcp-providers

Signed-off-by: srishtih <srishti.hegde.sh@gmail.com>

kranurag7 reviewed (commented):
I think a rebase to main would help us omit `mesa` deps which are part of diff now.

srishtih committed:
Merge branch 'main' into feat/gcp-providers

srishtih committed:
Clean up withdrawn-packages.txt

Removed multiple versions of mesa and onnx packages from the withdrawn-packages list.

Signed-off-by: srishtih <srishti.hegde.sh@gmail.com>
